.travis.yml

```diff
@@ -115,6 +124,11 @@ before_script:
   # prepare make files with cmake
   - |-
     case "${BUILD_TYPE}" in
+      "NonGitSources")
+        # test for NonGitSources
+        rm -rf ../.git
+        cmake ${GENERATOR} -DCMAKE_BUILD_TYPE=Debug -DENABLE_CODECOVERAGE=ON ..
```

Comment:
BTW, I know this is just a test build, but the `-DENABLE_CODECOVERAGE=ON` is causing a spurious failure (because this build isn't running the `before_install` steps needed to run coveralls). Other than that, the CI builds also seem okay? I wish I could help more, but I don't know what problem you are seeing.